### DISCUSSION TITLE
Question about cloud/remote object storage

### DISCUSSION THREAD

**Discussion by zulh-civo** [thanos-io/thanos#5543] [Questions & Answers]
Let's say I have 3 clusters like this:

![thanos](https://user-images.githubusercontent.com/75463191/181224246-1341881f-4b23-4a27-a1ee-9c1de807e070.png)

- Thanos: to run all Thanos components except Thanos Sidecar
- Cluster 1: to run Prometheus and Thanos Sidecar
- Cluster 2: to run Prometheus and Thanos Sidecar

Should I create 3 different cloud/remote buckets (e.g. S3 buckets) for each of them?

Should they share the same bucket, access key and secret key?

**Comment by krisztianfekete** [comment]
Hey, a single bucket is enough for you. 
iIf you have multiple tiers (dev, staging, prod), you can use multiple buckets per tier, but otherwise you are good with a single one.

**Reply by zulh-civo** [reply]
> Hey, a single bucket is enough for you. 
> 
> iIf you have multiple tiers (dev, staging, prod), you can use multiple buckets per tier, but otherwise you are good with a single one. 

Understood. Thank you.
